src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arr = [];
+  const newObj = Object.assign({}, state);
```

Comment:
don't use such naming (arr , obj)
you can use `stateVersions` and `stateCopy` or smth else semantic